### ISSUE TITLE
Tkinter label not showing

### ISSUE THREAD

**Issue by kfrankabh** [spyder-ide/spyder#21021]
### What steps will reproduce the problem?

import tkinter as tk
from tkinter import ttk

root = tk.Tk()
var = tk.StringVar()
var.set("initial")
label = ttk.Label(root, textvariable=var)
label.pack()
root.mainloop

When I run the above code in Spyder 5.4.3 running on Python 3.9.13, I get a blank Tk window. The Tkinter StringVar is not recognized. The above code runs in IDLE with no problem.

## Versions

* Spyder version: 5.4.3  (conda)
* Python version: 3.9.13 64-bit
* Qt version: 5.15.2
* PyQt5 version: 5.15.7
* Operating System: Windows 10

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0                             :  1.4.0 (OK)
chardet >=2.0.0                                  :  4.0.0 (OK)
cloudpickle >=0.5.0                              :  2.2.1 (OK)
cookiecutter >=1.6.0                             :  1.7.3 (OK)
diff_match_patch >=20181111                      :  20200713 (OK)
intervaltree >=3.0.2                             :  3.1.0 (OK)
IPython >=7.31.1,<9.0.0,!=8.8.0,!=8.9.0,!=8.10.0 :  8.12.0 (OK)
jedi >=0.17.2,<0.19.0                            :  0.18.1 (OK)
jellyfish >=0.7                                  :  0.9.0 (OK)
jsonschema >=3.2.0                               :  4.17.3 (OK)
keyring >=17.0.0                                 :  23.13.1 (OK)
nbconvert >=4.0                                  :  6.4.4 (OK)
numpydoc >=0.6.0                                 :  1.5.0 (OK)
paramiko >=2.4.0                                 :  None (NOK)
parso >=0.7.0,<0.9.0                             :  0.8.3 (OK)
pexpect >=4.4.0                                  :  4.8.0 (OK)
pickleshare >=0.4                                :  0.7.5 (OK)
psutil >=5.3                                     :  5.9.0 (OK)
pygments >=2.0                                   :  2.15.1 (OK)
pylint >=2.5.0,<3.0                              :  2.16.2 (OK)
pylint_venv >=2.1.1                              :  2.3.0 (OK)
pyls_spyder >=0.4.0                              :  0.4.0 (OK)
pylsp >=1.7.2,<1.8.0                             :  1.7.2 (OK)
pylsp_black >=1.2.0                              :  1.2.1 (OK)
qdarkstyle >=3.0.2,<3.2.0                        :  3.0.2 (OK)
qstylizer >=0.2.2                                :  0.2.2 (OK)
qtawesome >=1.2.1                                :  1.2.2 (OK)
qtconsole >=5.4.2,<5.5.0                         :  5.4.2 (OK)
qtpy >=2.1.0                                     :  2.2.0 (OK)
rtree >=0.9.7                                    :  1.0.1 (OK)
setuptools >=49.6.0                              :  67.8.0 (OK)
sphinx >=0.6.6                                   :  5.0.2 (OK)
spyder_kernels >=2.4.3,<2.5.0                    :  2.4.3 (OK)
textdistance >=4.2.0                             :  4.2.1 (OK)
three_merge >=0.1.1                              :  0.1.1 (OK)
watchdog >=0.10.3                                :  2.1.6 (OK)
zmq >=22.1.0                                     :  25.0.2 (OK)

# Optional:
cython >=0.21                                    :  None (NOK)
matplotlib >=3.0.0                               :  3.5.2 (OK)
numpy >=1.7                                      :  1.24.3 (OK)
pandas >=1.1.1                                   :  None (NOK)
scipy >=0.17.0                                   :  None (NOK)
sympy >=0.7.3                                    :  None (NOK)
```

### Environment

<details><summary>Environment</summary>

```
# packages in environment at C:\Users\Ken Frank\anaconda3\New folder\anaconda3:
#
# Name                    Version                   Build  Channel
_ipyw_jlab_nb_ext_conf    0.1.0            py39haa95532_0  
alabaster                 0.7.12             pyhd3eb1b0_0  
altgraph                  0.17.2                   pypi_0    pypi
anaconda-client           1.11.3           py39haa95532_0  
anaconda-navigator        2.3.0            py39haa95532_0  
anyio                     3.5.0            py39haa95532_0  
argon2-cffi               21.3.0             pyhd3eb1b0_0  
argon2-cffi-bindings      21.2.0           py39h2bbff1b_0  
arrow                     1.2.3            py39haa95532_1  
astroid                   2.14.2           py39haa95532_0  
asttokens                 2.0.5              pyhd3eb1b0_0  
atomicwrites              1.4.0                      py_0  
attrs                     22.1.0           py39haa95532_0  
autopep8                  1.6.0              pyhd3eb1b0_1  
babel                     2.11.0           py39haa95532_0  
backcall                  0.2.0              pyhd3eb1b0_0  
backports                 1.1                pyhd3eb1b0_0  
backports.functools_lru_cache 1.6.4              pyhd3eb1b0_0  
backports.tempfile        1.0                pyhd3eb1b0_1  
backports.weakref         1.0.post1                  py_1  
bcrypt                    3.2.0            py39h2bbff1b_1  
beautifulsoup4            4.12.2           py39haa95532_0  
binaryornot               0.4.4              pyhd3eb1b0_1  
black                     23.3.0           py39haa95532_0  
blas                      1.0                         mkl  
bleach                    4.1.0              pyhd3eb1b0_0  
brotli                    1.0.9                h2bbff1b_7  
brotli-bin                1.0.9                h2bbff1b_7  
brotlipy                  0.7.0           py39h2bbff1b_1003  
bzip2                     1.0.8                he774522_0  
ca-certificates           2023.05.30           haa95532_0  
certifi                   2023.5.7         py39haa95532_0  
cffi                      1.15.1           py39h2bbff1b_3  
chardet                   4.0.0           py39haa95532_1003  
charset-normalizer        2.0.4              pyhd3eb1b0_0  
click                     8.0.4            py39haa95532_0  
cloudpickle               2.2.1            py39haa95532_0  
clyent                    1.2.2            py39haa95532_1  
colorama                  0.4.6            py39haa95532_0  
comm                      0.1.2            py39haa95532_0  
conda                     22.9.0           py39haa95532_0  
conda-build               3.22.0           py39haa95532_0  
conda-content-trust       0.1.3            py39haa95532_0  
conda-env                 2.6.0                haa95532_1  
conda-package-handling    2.1.0            py39haa95532_0  
conda-package-streaming   0.8.0            py39haa95532_0  
conda-repo-cli            1.0.41           py39haa95532_0  
conda-token               0.4.0              pyhd3eb1b0_0  
conda-verify              3.4.2                      py_1  
console_shortcut          0.1.1                         4  
cookiecutter              1.7.3              pyhd3eb1b0_0  
cryptography              39.0.1           py39h21b164f_1  
cycler                    0.11.0             pyhd3eb1b0_0  
debugpy                   1.5.1            py39hd77b12b_0  
decorator                 5.1.1              pyhd3eb1b0_0  
defusedxml                0.7.1              pyhd3eb1b0_0  
diff-match-patch          20200713           pyhd3eb1b0_0  
dill                      0.3.6            py39haa95532_0  
docstring-to-markdown     0.11             py39haa95532_0  
docutils                  0.18.1           py39haa95532_3  
entrypoints               0.4              py39haa95532_0  
executing                 0.8.3              pyhd3eb1b0_0  
filelock                  3.9.0            py39haa95532_0  
flake8                    6.0.0            py39haa95532_0  
fonttools                 4.25.0             pyhd3eb1b0_0  
freetype                  2.12.1               ha860e81_0  
future                    0.18.3           py39haa95532_0  
giflib                    5.2.1                h8cc25b3_3  
glib                      2.69.1               h5dc1a3c_2  
glob2                     0.7                pyhd3eb1b0_0  
gst-plugins-base          1.18.5               h9e645db_0  
gstreamer                 1.18.5               hd78058f_0  
icu                       58.2                 ha925a31_3  
idna                      3.4              py39haa95532_0  
imagesize                 1.4.1            py39haa95532_0  
importlib-metadata        6.0.0            py39haa95532_0  
importlib_metadata        6.0.0                hd3eb1b0_0  
inflection                0.5.1            py39haa95532_0  
intel-openmp              2023.1.0         h59b6b97_46319  
intervaltree              3.1.0              pyhd3eb1b0_0  
ipykernel                 6.19.2           py39hd4e2768_0  
ipython                   8.12.0           py39haa95532_0  
ipython_genutils          0.2.0              pyhd3eb1b0_1  
ipywidgets                8.0.4            py39haa95532_0  
isort                     5.9.3              pyhd3eb1b0_0  
jaraco.classes            3.2.1              pyhd3eb1b0_0  
jedi                      0.18.1           py39haa95532_1  
jellyfish                 0.9.0            py39h2bbff1b_0  
jinja2                    2.11.3             pyhd3eb1b0_0  
jinja2-time               0.2.0              pyhd3eb1b0_3  
jpeg                      9e                   h2bbff1b_1  
json5                     0.9.6              pyhd3eb1b0_0  
jsonschema                4.17.3           py39haa95532_0  
jupyter_client            8.1.0            py39haa95532_0  
jupyter_core              5.3.0            py39haa95532_0  
jupyter_server            1.23.4           py39haa95532_0  
jupyterlab                3.5.3            py39haa95532_0  
jupyterlab_pygments       0.1.2                      py_0  
jupyterlab_server         2.10.3             pyhd3eb1b0_1  
jupyterlab_widgets        3.0.5            py39haa95532_0  
keyring                   23.13.1          py39haa95532_0  
kiwisolver                1.4.4            py39hd77b12b_0  
krb5                      1.19.4               h5b6d351_0  
lazy-object-proxy         1.6.0            py39h2bbff1b_0  
lerc                      3.0                  hd77b12b_0  
libarchive                3.6.2                h2033e3e_1  
libbrotlicommon           1.0.9                h2bbff1b_7  
libbrotlidec              1.0.9                h2bbff1b_7  
libbrotlienc              1.0.9                h2bbff1b_7  
libclang                  14.0.6          default_hb5a9fac_1  
libclang13                14.0.6          default_h8e68704_1  
libdeflate                1.17                 h2bbff1b_0  
libffi                    3.4.4                hd77b12b_0  
libiconv                  1.16                 h2bbff1b_2  
liblief                   0.12.3               hd77b12b_0  
libogg                    1.3.5                h2bbff1b_1  
libpng                    1.6.39               h8cc25b3_0  
libsodium                 1.0.18               h62dcd97_0  
libspatialindex           1.9.3                h6c2663c_0  
libtiff                   4.5.0                h6c2663c_2  
libvorbis                 1.3.7                he774522_0  
libwebp                   1.2.4                hbc33d0d_1  
libwebp-base              1.2.4                h2bbff1b_1  
libxml2                   2.10.3               h0ad7f3c_0  
libxslt                   1.1.37               h2bbff1b_0  
lxml                      4.9.2            py39h2bbff1b_0  
lz4-c                     1.9.4                h2bbff1b_0  
m2-msys2-runtime          2.5.0.17080.65c939c               3  
m2-patch                  2.7.5                         2  
markupsafe                2.0.1            py39h2bbff1b_0  
matplotlib                3.5.2            py39haa95532_0  
matplotlib-base           3.5.2            py39hd77b12b_0  
matplotlib-inline         0.1.6            py39haa95532_0  
mccabe                    0.7.0              pyhd3eb1b0_0  
menuinst                  1.4.19           py39h59b6b97_0  
mistune                   0.8.4           py39h2bbff1b_1000  
mkl                       2023.1.0         h8bd8f75_46356  
mkl-service               2.4.0            py39h2bbff1b_1  
mkl_fft                   1.3.6            py39hf11a4ad_1  
mkl_random                1.2.2            py39hf11a4ad_1  
more-itertools            8.12.0             pyhd3eb1b0_0  
msys2-conda-epoch         20160418                      1  
munkres                   1.1.4                      py_0  
mypy_extensions           0.4.3            py39haa95532_1  
navigator-updater         0.3.0            py39haa95532_0  
nbclassic                 0.5.5            py39haa95532_0  
nbclient                  0.5.13           py39haa95532_0  
nbconvert                 6.4.4            py39haa95532_0  
nbformat                  5.7.0            py39haa95532_0  
nest-asyncio              1.5.6            py39haa95532_0  
notebook                  6.5.4            py39haa95532_0  
notebook-shim             0.2.2            py39haa95532_0  
numpy                     1.24.3           py39h79a8e48_1  
numpy-base                1.24.3           py39h8a87ada_1  
numpydoc                  1.5.0            py39haa95532_0  
openssl                   1.1.1t               h2bbff1b_0  
packaging                 23.0             py39haa95532_0  
pandocfilters             1.5.0              pyhd3eb1b0_0  
paramiko                  2.8.1              pyhd3eb1b0_0  
parso                     0.8.3              pyhd3eb1b0_0  
pathlib                   1.0.1              pyhd3eb1b0_1  
pathspec                  0.10.3           py39haa95532_0  
pcre                      8.45                 hd77b12b_0  
pefile                    2022.5.30                pypi_0    pypi
pexpect                   4.8.0              pyhd3eb1b0_3  
pickleshare               0.7.5           pyhd3eb1b0_1003  
pillow                    9.4.0            py39hd77b12b_0  
pip                       23.0.1           py39haa95532_0  
pkginfo                   1.9.6            py39haa95532_0  
platformdirs              2.5.2            py39haa95532_0  
pluggy                    1.0.0            py39haa95532_1  
ply                       3.11             py39haa95532_0  
powershell_shortcut       0.0.1                         3  
poyo                      0.5.0              pyhd3eb1b0_0  
prometheus_client         0.14.1           py39haa95532_0  
prompt-toolkit            3.0.36           py39haa95532_0  
psutil                    5.9.0            py39h2bbff1b_0  
ptyprocess                0.7.0              pyhd3eb1b0_2  
pure_eval                 0.2.2              pyhd3eb1b0_0  
py-lief                   0.12.3           py39hd77b12b_0  
pycodestyle               2.10.0           py39haa95532_0  
pycosat                   0.6.4            py39h2bbff1b_0  
pycparser                 2.21               pyhd3eb1b0_0  
pydocstyle                6.3.0            py39haa95532_0  
pyflakes                  3.0.1            py39haa95532_0  
pygments                  2.15.1           py39haa95532_1  
pyinstaller               5.4.1                    pypi_0    pypi
pyinstaller-hooks-contrib 2022.10                  pypi_0    pypi
pyjwt                     2.4.0            py39haa95532_0  
pylint                    2.16.2           py39haa95532_0  
pylint-venv               2.3.0            py39haa95532_0  
pyls-spyder               0.4.0              pyhd3eb1b0_0  
pynacl                    1.5.0            py39h8cc25b3_0  
pyopenssl                 23.0.0           py39haa95532_0  
pyparsing                 3.0.9            py39haa95532_0  
pyqt                      5.15.7           py39hd77b12b_0  
pyqt5-sip                 12.11.0          py39hd77b12b_0  
pyqtwebengine             5.15.7           py39hd77b12b_0  
pyrsistent                0.18.0           py39h196d8e1_0  
pysimplegui               4.60.3                   pypi_0    pypi
pysimplegui-exemaker      1.3.0                    pypi_0    pypi
pysocks                   1.7.1            py39haa95532_0  
python                    3.9.13               h6244533_1  
python-dateutil           2.8.2              pyhd3eb1b0_0  
python-docx               0.8.11             pyhd8ed1ab_0    conda-forge
python-fastjsonschema     2.16.2           py39haa95532_0  
python-libarchive-c       2.9                pyhd3eb1b0_1  
python-lsp-black          1.2.1            py39haa95532_0  
python-lsp-jsonrpc        1.0.0              pyhd3eb1b0_0  
python-lsp-server         1.7.2            py39haa95532_0  
python-slugify            5.0.2              pyhd3eb1b0_0  
pytoolconfig              1.2.5            py39haa95532_1  
pytz                      2022.7           py39haa95532_0  
pywin32                   305              py39h2bbff1b_0  
pywin32-ctypes            0.2.0           py39haa95532_1000  
pywinpty                  2.0.10           py39h5da7b33_0  
pyyaml                    6.0              py39h2bbff1b_1  
pyzmq                     25.0.2           py39hd77b12b_0  
qdarkstyle                3.0.2              pyhd3eb1b0_0  
qstylizer                 0.2.2            py39haa95532_0  
qt-main                   5.15.2               he8e5bd7_8  
qt-webengine              5.15.9               hb9a9bb5_5  
qtawesome                 1.2.2            py39haa95532_0  
qtconsole                 5.4.2            py39haa95532_0  
qtpy                      2.2.0            py39haa95532_0  
qtwebkit                  5.212                h2bbfb41_5  
requests                  2.29.0           py39haa95532_0  
requests-toolbelt         0.9.1              pyhd3eb1b0_0  
rope                      1.7.0            py39haa95532_0  
rtree                     1.0.1            py39h2eaa2aa_0  
ruamel_yaml               0.15.100         py39h2bbff1b_0  
scp                       0.14.1             pyhd3eb1b0_0  
send2trash                1.8.0              pyhd3eb1b0_1  
setuptools                67.8.0           py39haa95532_0  
sip                       6.6.2            py39hd77b12b_0  
six                       1.16.0             pyhd3eb1b0_1  
sniffio                   1.2.0            py39haa95532_1  
snowballstemmer           2.2.0              pyhd3eb1b0_0  
sortedcontainers          2.4.0              pyhd3eb1b0_0  
soupsieve                 2.4              py39haa95532_0  
sphinx                    5.0.2            py39haa95532_0  
sphinxcontrib-applehelp   1.0.2              pyhd3eb1b0_0  
sphinxcontrib-devhelp     1.0.2              pyhd3eb1b0_0  
sphinxcontrib-htmlhelp    2.0.0              pyhd3eb1b0_0  
sphinxcontrib-jsmath      1.0.1              pyhd3eb1b0_0  
sphinxcontrib-qthelp      1.0.3              pyhd3eb1b0_0  
sphinxcontrib-serializinghtml 1.1.5              pyhd3eb1b0_0  
spyder                    5.4.3            py39haa95532_1  
spyder-kernels            2.4.3            py39haa95532_0  
sqlite                    3.41.2               h2bbff1b_0  
stack_data                0.2.0              pyhd3eb1b0_0  
tbb                       2021.8.0             h59b6b97_0  
terminado                 0.17.1           py39haa95532_0  
testpath                  0.6.0            py39haa95532_0  
text-unidecode            1.3                pyhd3eb1b0_0  
textdistance              4.2.1              pyhd3eb1b0_0  
three-merge               0.1.1              pyhd3eb1b0_0  
tinycss2                  1.2.1            py39haa95532_0  
tk                        8.6.12               h2bbff1b_0  
toml                      0.10.2             pyhd3eb1b0_0  
tomli                     2.0.1            py39haa95532_0  
tomlkit                   0.11.1           py39haa95532_0  
toolz                     0.12.0           py39haa95532_0  
tornado                   6.2              py39h2bbff1b_0  
tqdm                      4.65.0           py39hd4e2768_0  
traitlets                 5.7.1            py39haa95532_0  
typing-extensions         4.5.0            py39haa95532_0  
typing_extensions         4.5.0            py39haa95532_0  
tzdata                    2023c                h04d1e81_0  
ujson                     5.4.0            py39hd77b12b_0  
unidecode                 1.2.0              pyhd3eb1b0_0  
urllib3                   1.26.16          py39haa95532_0  
vc                        14.2                 h21ff451_1  
vs2015_runtime            14.27.29016          h5e58377_2  
watchdog                  2.1.6            py39haa95532_0  
wcwidth                   0.2.5              pyhd3eb1b0_0  
webencodings              0.5.1            py39haa95532_1  
websocket-client          0.58.0           py39haa95532_4  
whatthepatch              1.0.2            py39haa95532_0  
wheel                     0.38.4           py39haa95532_0  
widgetsnbextension        4.0.5            py39haa95532_0  
win_inet_pton             1.1.0            py39haa95532_0  
winpty                    0.4.3                         4  
wrapt                     1.14.1           py39h2bbff1b_0  
xz                        5.4.2                h8cc25b3_0  
yaml                      0.2.5                he774522_0  
yapf                      0.31.0             pyhd3eb1b0_0  
zeromq                    4.3.4                hd77b12b_0  
zipp                      3.11.0           py39haa95532_0  
zlib                      1.2.13               h8cc25b3_0  
zstandard                 0.19.0           py39h2bbff1b_0  
zstd                      1.5.5                hd43e919_0  

```
</details>

**Comment by dalthviz** [comment]
Hi @kfrankabh thank you for the feedback! Could it be possible for you to add parenthesis to the last line of your code? So instead of `root.mainloop` using `root.mainloop()`? Checked locally and seems like that enables showing the window with the label:

![image](https://github.com/spyder-ide/spyder/assets/16781833/d0ada036-e8c1-4144-ae90-0832b2de93b3)

Let us know if the suggestion above helps!

**Comment by kfrankabh** [comment]
Thank you for your prompt and kindly worded suggestion. I added the
parentheses to root.mainloop, but I the Label widget still does not display
the textvariable:

[image: image.png]

I'm wondering about a message that appears when I open Spyder:

[image: image.png]

I am launching Spyder 5.4.3 from Anaconda Navigator. The installed package
for Paramiko is version 2.8.1, so I don't know why I'm getting this error
message.
I also wonder whether this error has anything to do with my Tkinter
textvariable problem.

I surely appreciate your help!

--Ken

==============

**Comment by dalthviz** [comment]
@kfrankabh seems like you attached some images in your response. However, since you replied via email, those images are not visible in the GitHub interface. Could you post the images/screenshots directly in the GitHub issue please? 

Regarding the `paramiko` message, could it be related to https://github.com/spyder-ide/spyder/issues/21022 ?

Did you install or updated Spyder or you just installed the Anaconda distribution and started Spyder to see the message?

Not totally sure what could be happening but maybe something is working strangely for Spyder installations that use the default conda channel/come installed in the anaconda base environment?

Any other info to understand the issue with tkinter is appreciated! And in relation with the `paramiko` message, please give a check to the linked issue and post any other related info there!

**Comment by kfrankabh** [comment]
Thank you for your prompt and kindly worded suggestion. I added the parentheses to root.mainloop, but I the Label widget still does not display the textvariable:

![image](https://github.com/spyder-ide/spyder/assets/136278263/79f7cf23-271b-4bb4-ac9d-63ece8e7b96e)

I'm wondering about a message that appears when I open Spyder:

![image](https://github.com/spyder-ide/spyder/assets/136278263/67ce59e1-a23f-4cab-acdf-9b27ce72727e)

[Images TkinterWindow ParamikoError.pdf](https://github.com/spyder-ide/spyder/files/11737191/Images.TkinterWindow.ParamikoError.pdf)

I am launching Spyder 5.4.3 from Anaconda Navigator. The installed package for Paramiko is version 2.8.1, so I don't know why I'm getting this error message.

I also wonder whether this error has anything to do with my Tkinter textvariable problem.

I surely appreciate your help!

Note: I have tried to put my images (my Tkinter window, and the Paramiko error message) in this post by pasting screenshots directly, and also by attaching a pdf. I can't tell if I've been successful.

Issue 21022 is exactly the error message I get, but I don't know enough to understand what I'm seeing.

--Ken

**Comment by dalthviz** [comment]
Thank you for submitting here the screenshots @kfrankabh ! I think something we could try to do, to be sure the problems you are having are not installation related, is to create a new environment with Spyder in it. For that, you will need to open an Anaconda prompt and run the following:

    conda create -n spyder-df python=3.9.13 spyder
    conda activate spyder-df
    spyder

With the commands above you will create an env called `spyder-df` with Python 3.9.13 and Spyder installed from the conda default packages channel in it, activate the created env and finally launch Spyder from it.

Let us know if using Spyder from the new env prevents the empty tkinter window and the `paramiko` message!

**Comment by kfrankabh** [comment]
I created the new environment with Spyder in it. I ran the Tkinter scripts, the tkinter label window is still empty:

![image](https://github.com/spyder-ide/spyder/assets/136278263/7d0c9124-08e2-4060-b7a2-e879677029d3)

When I launched Spyder the first time from the Anaconda prompt, the paramiko message did not appear. But when I launched it a second time, I got the message:

![image](https://github.com/spyder-ide/spyder/assets/136278263/edfb3e58-7937-4156-9f69-7ade6a45e384)

I hope you might have some idea of what's happening.

Thanks again for all of your kind help.

--Ken

**Comment by dalthviz** [comment]
I see, I checked locally but I'm still unable to reproduce this :/

Also, checking again the environment description in the OP, seems like you installed anaconda inside anaconda? Your base env path seems a little bit strange: `C:\Users\Ken Frank\anaconda3\New folder\anaconda3`

Have you recently tried to reinstall anaconda without uninstalling a previous installation first? Is there a possibility that you could uninstall anaconda and reinstall it again? 

Besides checking your anaconda installation another thing you could try is to create and env but using the conda-forge channel to get the packages running from an anaconda prompt something like:

    conda create -n spyder-cf python=3.9.13 spyder
    conda activate spyder-cf
    spyder

Also, to launch Spyder (and be sure you are using the Spyder installation from a specific environment) you can search from your start menu something like `Spyder (<your env name>)`. So for example to launch Spyder from the `spyder-cf` env tou can serach for `Spyder (spyder-cf)`.

Let us know if something of the above helps!

**Comment by kfrankabh** [comment]
You did it! Brilliant! Thank you so much.
I uninstalled Anaconda correctly and then reinstalled it.
Then I created a Spyder environment using the anaconda prompt.
Then I used the spyder-cf shortcut to open Spyder. It works perfectly with my Tkinter scripts--no errors. Wonderful!

Thank you so much for your patience and insight.

(You might like to know that when I launch Spyder using the Anaconda Navigator application tile, it gives the same errors as before. So I won't be doing that.)
